src/root/rmem.d

```diff
@@ -194,6 +194,14 @@ else
             return cast(Object)p;
         }
 
+      version (LDC)
```

Comment:
Funky in the sense of emphasizing an LDC-specific addition in the same scope with an equal indentation level, yep, guilty. ;) Updated to prevent further discussions.